docs/release-notes/NuGet-6.10.md

```diff
@@ -12,7 +12,17 @@ NuGet distribution vehicles:
 
 | NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
 |:---|:---|:---|
-| [**6.10.1**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.10](https://visualstudio.microsoft.com/downloads/) | [8.0.3xx](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+| [**6.10.2**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.10](https://visualstudio.microsoft.com/downloads/) | [8.0.3xx](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.10.2
+
+### Issues fixed in this release
+
+* Revert Enable central package management implicitly when Directory.Packages.props exists - [#11834](https://github.com/NuGet/Home/issues/11834)
+
+[List of commits in this release](https://github.com/NuGet/NuGet.Client/compare/6.10.1.5...release-6.10.x)
```

Comment:
The URL should use the tag name/label, not the branch name. Otherwise, if we add more changes to the branch, it will incorrectly list those changes when the customer clicks the link "in this release"